packages/web-components/src/google-map/components/google-map.svelte

```diff
@@ -0,0 +1,42 @@
+<script lang="typescript">
+  import { onMount, createEventDispatcher } from 'svelte'
+  import store from '../core/store'
+  import GoogleMapLoader from './google-map-loader.svelte'
+  import * as Styles from '../../common/styles'
+
+  const dispatch = createEventDispatcher()
+  const { resetCSS } = Styles
+
+  export let apiKey: string = ''
+  export let libraries: string = 'places'
+  export let center = null
+  export let zoom = 10
+  export let onLoaded = null
+
+  let mapElement
+  let map
+
+  const initialise = () => {
+    setTimeout(() => {
```

Comment:
Not sure why this needs to be in a timeout - can this component not subscribe to a store and listen on a map-loaded value?